Makefile

```diff
@@ -28,6 +35,10 @@ run: | $(grpc_services_directory)
 docker-build:
 	docker build -f Dockerfile-local --force-rm -t quay.io/team-helium/router:local .
 
+docker-docs:
```

Comment:
Will this extract the created file back to local env?

Reply:
Yes, the generated .png and .svg files will appear on the local laptop/workstation file system because `-v` (mount local volume) is given to `docker run` without needing to extract or copy any files.